### ISSUE TITLE
lab report template leading to errors: duplicate chunk labels

### ISSUE THREAD

**Issue by mamcisaac** [OpenIntroStat/oilabs-tidy#110]
The Lab Report template provided in the openintro package has a sample code chunk for Exercise 1 and that code chunk has a label ("code-chunk-label"). 

Many of my students copy and paste this code chunk when completing later Exercises, which leads to an error caused by duplicate code chunk labels. This issue is well outside the scope of what I want them to be considering in an introductory class. 

A lot of unnecessary confusion could be avoided if the template simply didn't have the "code-chunk-label" label in that first code chunk. 

Similarly, the ellipsis at the bottom of the document (presumably meant to indicate to students that they should continue the document after Exercise 2) seems to cause confusion to students new to RMarkdown because many assume it is an important part of the document, like the chunk delimiters ("```" vs "..."). 

To summarize, I recommend avoiding unnecessary confusion by modifying the Lab Report template by 

1. deleting the code chunk label "code-chunk-label"
2. deleting the ellipsis ("...") at the bottom of the document
